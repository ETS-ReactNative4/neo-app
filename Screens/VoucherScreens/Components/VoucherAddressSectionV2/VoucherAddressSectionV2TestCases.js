import React from "react";
import VoucherAddressSectionV2 from "./VoucherAddressSectionV2";

const VoucherAddressSectionV2TestCases = [
  {
    title: "Voucher Address Section",
    Component: (
      <VoucherAddressSectionV2
        address={
          "Jl. Benesari No.2, Kuta, Kabupaten Badung, Bali 80361, Indonesia"
        }
      />
    )
  }
];

export default VoucherAddressSectionV2TestCases;
