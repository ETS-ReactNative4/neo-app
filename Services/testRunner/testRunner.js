import renderer from "react-test-renderer";

const testRunner = testCase => {
  test(testCase.title, () => {
    const tree = renderer.create(testCase.Component).toJSON();
    expect(tree).toMatchSnapshot();
  });
};

export default testRunner;
