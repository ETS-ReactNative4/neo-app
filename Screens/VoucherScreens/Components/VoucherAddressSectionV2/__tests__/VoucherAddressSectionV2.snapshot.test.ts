import VoucherAddressSectionV2TestCases from "../VoucherAddressSectionV2TestCases";
import testRunner from "../../../../../Services/testRunner/testRunner";

VoucherAddressSectionV2TestCases.map(testRunner);
