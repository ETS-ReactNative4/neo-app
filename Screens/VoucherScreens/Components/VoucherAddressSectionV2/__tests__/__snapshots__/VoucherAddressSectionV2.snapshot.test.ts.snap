// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Voucher Address Section 1`] = `
<View
  style={
    Array [
      Object {
        "padding": 24,
      },
      Object {},
    ]
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "flex-start",
      }
    }
  >
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": undefined,
            "fontSize": 14,
          },
          undefined,
          Object {
            "fontFamily": "voyager",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      î¤”
    </Text>
    <Text
      style={
        Object {
          "color": "rgba(44, 47, 66, 1)",
          "fontFamily": "SourceSansPro-SemiBold",
          "fontSize": 14,
          "lineHeight": 14,
          "marginLeft": 8,
          "marginTop": 2,
        }
      }
    >
      ADDRESS
    </Text>
  </View>
  <View
    style={
      Object {
        "marginTop": 16,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "rgba(83, 87, 109, 1)",
          "fontFamily": "SourceSansPro-Regular",
          "fontSize": 14,
          "lineHeight": 14,
        }
      }
    >
      Jl. Benesari No.2, Kuta, Kabupaten Badung, Bali 80361, Indonesia
    </Text>
  </View>
</View>
`;
